VERSION

```diff
@@ -0,0 +1,2 @@
+RIOT_VERSION = 2024.04
+
```

Comment:
```suggestion
```

Reply:
Hehe, the noobie error. I abandoned VS Code recenetnly, guess you can tell from this oversight.